### PR TITLE
Bump jruby

### PR DESCRIPTION
**Pull request by cpuguy83**
Also removes arm32-alpine builds which are failing due to issues in jruby.

### CODE REVIEW

File: library/jruby [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+GitCommit: a9179975e60a96a6e9d880bcf2dd566ab290b539
 
-Tags: latest, 9, 9.1, 9.1-jre, 9.1.13, 9.1.13-jre, 9.1.13.0, 9.1.13.0-jre
+Tags: latest, 9, 9.1, 9.1.13, 9.1-jre, 9.1.13-jre, 9.1.13.0, 9.1.13.0-jre
 Architectures: amd64, i386, arm32v5, arm64v8, ppc64le, s390x
 Directory: 9000/jre
 
-Tags: 9-alpine, 9.1-alpine, 9.1-jre-alpine, 9.1.13-alpine, 9.1.13-jre-alpine, 9.1.13.0-alpine, 9.1.13.0-jre-alpine
-Architectures: amd64, arm32v6, arm64v8, ppc64le, s390x
+Tags: 9-alpine, 9.1-alpine, 9.1.13-alpine, 9-alpine, 9.1-jre-alpine, 9.1.13-jre-alpine, 9.1.13.0-alpine, 9.1.13.0-jre-alpine
```
Comment (tianon) [inline library/jruby]: `9-alpine` is in here twice :smile:

Comment (cpuguy83) [reply library/jruby]: Also another issue, but you'll never find it now :)

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index da5b8db..9876503 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -16,31 +16,29 @@ jruby:1.7.27-onbuild @ arm64v8
 jruby:1.7.27-onbuild @ i386
 jruby:1.7.27-onbuild @ ppc64le
 jruby:1.7.27-onbuild @ s390x
-jruby:9.1.13.0-jdk @ amd64
-jruby:9.1.13.0-jdk @ arm32v5
-jruby:9.1.13.0-jdk @ arm64v8
-jruby:9.1.13.0-jdk @ i386
-jruby:9.1.13.0-jdk @ ppc64le
-jruby:9.1.13.0-jdk @ s390x
-jruby:9.1.13.0-jdk-alpine @ amd64
-jruby:9.1.13.0-jdk-alpine @ arm32v6
-jruby:9.1.13.0-jdk-alpine @ arm64v8
-jruby:9.1.13.0-jdk-alpine @ ppc64le
-jruby:9.1.13.0-jdk-alpine @ s390x
-jruby:9.1.13.0-jre @ amd64
-jruby:9.1.13.0-jre @ arm32v5
-jruby:9.1.13.0-jre @ arm64v8
-jruby:9.1.13.0-jre @ i386
-jruby:9.1.13.0-jre @ ppc64le
-jruby:9.1.13.0-jre @ s390x
-jruby:9.1.13.0-jre-alpine @ amd64
-jruby:9.1.13.0-jre-alpine @ arm32v6
-jruby:9.1.13.0-jre-alpine @ arm64v8
-jruby:9.1.13.0-jre-alpine @ ppc64le
-jruby:9.1.13.0-jre-alpine @ s390x
-jruby:9.1.13.0-onbuild @ amd64
-jruby:9.1.13.0-onbuild @ arm32v5
-jruby:9.1.13.0-onbuild @ arm64v8
-jruby:9.1.13.0-onbuild @ i386
-jruby:9.1.13.0-onbuild @ ppc64le
-jruby:9.1.13.0-onbuild @ s390x
+jruby:9.1.14.0-jdk @ amd64
+jruby:9.1.14.0-jdk @ arm32v5
+jruby:9.1.14.0-jdk @ arm64v8
+jruby:9.1.14.0-jdk @ i386
+jruby:9.1.14.0-jdk @ ppc64le
+jruby:9.1.14.0-jdk @ s390x
+jruby:9.1.14.0-jdk-alpine @ amd64
+jruby:9.1.14.0-jdk-alpine @ arm64v8
+jruby:9.1.14.0-jdk-alpine @ ppc64le
+jruby:9.1.14.0-jdk-alpine @ s390x
+jruby:9.1.14.0-jre @ amd64
+jruby:9.1.14.0-jre @ arm32v5
+jruby:9.1.14.0-jre @ arm64v8
+jruby:9.1.14.0-jre @ i386
+jruby:9.1.14.0-jre @ ppc64le
+jruby:9.1.14.0-jre @ s390x
+jruby:9.1.14.0-jre-alpine @ amd64
+jruby:9.1.14.0-jre-alpine @ arm64v8
+jruby:9.1.14.0-jre-alpine @ ppc64le
+jruby:9.1.14.0-jre-alpine @ s390x
+jruby:9.1.14.0-onbuild @ amd64
+jruby:9.1.14.0-onbuild @ arm32v5
+jruby:9.1.14.0-onbuild @ arm64v8
+jruby:9.1.14.0-onbuild @ i386
+jruby:9.1.14.0-onbuild @ ppc64le
+jruby:9.1.14.0-onbuild @ s390x
diff --git a/_bashbrew-list b/_bashbrew-list
index fde1faf..286d380 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -8,6 +8,8 @@ jruby:1.7.27-jre
 jruby:1.7.27-onbuild
 jruby:9
 jruby:9-alpine
+jruby:9-jdk
+jruby:9-jdk-alpine
 jruby:9-onbuild
 jruby:9.1
 jruby:9.1-alpine
@@ -16,18 +18,18 @@ jruby:9.1-jdk-alpine
 jruby:9.1-jre
 jruby:9.1-jre-alpine
 jruby:9.1-onbuild
-jruby:9.1.13
-jruby:9.1.13-alpine
-jruby:9.1.13-jdk
-jruby:9.1.13-jdk-alpine
-jruby:9.1.13-jre
-jruby:9.1.13-jre-alpine
-jruby:9.1.13-onbuild
-jruby:9.1.13.0
-jruby:9.1.13.0-alpine
-jruby:9.1.13.0-jdk
-jruby:9.1.13.0-jdk-alpine
-jruby:9.1.13.0-jre
-jruby:9.1.13.0-jre-alpine
-jruby:9.1.13.0-onbuild
+jruby:9.1.14
+jruby:9.1.14-alpine
+jruby:9.1.14-jdk
+jruby:9.1.14-jdk-alpine
+jruby:9.1.14-jre
+jruby:9.1.14-jre-alpine
+jruby:9.1.14-onbuild
+jruby:9.1.14.0
+jruby:9.1.14.0-alpine
+jruby:9.1.14.0-jdk
+jruby:9.1.14.0-jdk-alpine
+jruby:9.1.14.0-jre
+jruby:9.1.14.0-jre-alpine
+jruby:9.1.14.0-onbuild
 jruby:latest
diff --git a/jruby_9.1.13.0-jdk-alpine/Dockerfile b/jruby_9.1.14.0-jdk-alpine/Dockerfile
similarity index 92%
rename from jruby_9.1.13.0-jdk-alpine/Dockerfile
rename to jruby_9.1.14.0-jdk-alpine/Dockerfile
index 7cd0fde..5e66b85 100644
--- a/jruby_9.1.13.0-jdk-alpine/Dockerfile
+++ b/jruby_9.1.14.0-jdk-alpine/Dockerfile
@@ -4,8 +4,8 @@ RUN apk add --no-cache \
       bash \
       libc6-compat
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 
 RUN apk add --no-cache --virtual .build-deps \
       curl \
diff --git a/jruby_9.1.13.0-jdk/Dockerfile b/jruby_9.1.14.0-jdk/Dockerfile
similarity index 91%
rename from jruby_9.1.13.0-jdk/Dockerfile
rename to jruby_9.1.14.0-jdk/Dockerfile
index da9e5ad..797b929 100644
--- a/jruby_9.1.13.0-jdk/Dockerfile
+++ b/jruby_9.1.14.0-jdk/Dockerfile
@@ -2,8 +2,8 @@ FROM openjdk:8-jdk
 
 RUN apt-get update && apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 RUN mkdir /opt/jruby \
   && curl -fSL https://s3.amazonaws.com/jruby.org/downloads/${JRUBY_VERSION}/jruby-bin-${JRUBY_VERSION}.tar.gz -o /tmp/jruby.tar.gz \
   && echo "$JRUBY_SHA256 /tmp/jruby.tar.gz" | sha256sum -c - \
diff --git a/jruby_9.1.13.0-jre-alpine/Dockerfile b/jruby_9.1.14.0-jre-alpine/Dockerfile
similarity index 89%
rename from jruby_9.1.13.0-jre-alpine/Dockerfile
rename to jruby_9.1.14.0-jre-alpine/Dockerfile
index 510c6b7..a52d1fb 100644
--- a/jruby_9.1.13.0-jre-alpine/Dockerfile
+++ b/jruby_9.1.14.0-jre-alpine/Dockerfile
@@ -1,11 +1,11 @@
-FROM openjdk:8-jre-alpine
+FROM i386/openjdk:8-jre-alpine
 
 RUN apk add --no-cache \
       bash \
       libc6-compat
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 
 RUN apk add --no-cache --virtual .build-deps \
       curl \
diff --git a/jruby_9.1.13.0-jre/Dockerfile b/jruby_9.1.14.0-jre/Dockerfile
similarity index 91%
rename from jruby_9.1.13.0-jre/Dockerfile
rename to jruby_9.1.14.0-jre/Dockerfile
index 2283b9c..0f7147e 100644
--- a/jruby_9.1.13.0-jre/Dockerfile
+++ b/jruby_9.1.14.0-jre/Dockerfile
@@ -2,8 +2,8 @@ FROM openjdk:8-jre
 
 RUN apt-get update && apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 RUN mkdir /opt/jruby \
   && curl -fSL https://s3.amazonaws.com/jruby.org/downloads/${JRUBY_VERSION}/jruby-bin-${JRUBY_VERSION}.tar.gz -o /tmp/jruby.tar.gz \
   && echo "$JRUBY_SHA256 /tmp/jruby.tar.gz" | sha256sum -c - \
diff --git a/jruby_9.1.13.0-onbuild/Dockerfile b/jruby_9.1.14.0-onbuild/Dockerfile
similarity index 100%
rename from jruby_9.1.13.0-onbuild/Dockerfile
rename to jruby_9.1.14.0-onbuild/Dockerfile
````

</details>

**Comment by yosifkit** [comment]
```
+FROM i386/openjdk:8-jre-alpine
```
:exclamation: :see_no_evil:

**Comment by tianon** [comment]
@cpuguy83 given https://github.com/jruby/jruby/issues/4112#issuecomment-318607095, what're your thoughts on removing the 1.7 versions? :innocent:

(Removing the tags here will remove them from the "Supported" section on the Hub readme and will prevent us from spending cycles rebuilding it on the official build server, but the tag will still be available to users who want it as in https://github.com/docker-library/official-images#library-definition-files.)

**Comment by yosifkit** [comment]
```console
testing jruby:1.7-jdk
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...passed
	'override-cmd' [4/9]...passed
	'ruby-hello-world' [5/9]...passed
	'ruby-standard-libs' [6/9]...passed
	'ruby-gems' [7/9]...ERROR:  Error installing rake:
	rake requires Ruby version >= 2.0.0.
failed
	'ruby-bundler' [8/9]...passed
	'ruby-nonroot' [9/9]...passed
```

So, `rake` now requires ruby `2.0` (https://github.com/ruby/rake/pull/236). Should we change our [test](https://github.com/docker-library/official-images/blob/0f280c6cb1baaf88b34fb7fcd949d9ee247b84e5/test/tests/ruby-gems/container.sh) to account for ruby less than `2.0` or remove jruby `1.7`?

(also, the `FROM i386/` needs fixing :arrow_up:)

**Comment by tianon** [comment]
Test updated: https://github.com/docker-library/official-images/pull/3720

Just need to decide whether you're keeping 1.7 for now and fix the `i386`. :muscle:

**Comment by cpuguy83** [comment]
Fixed the i386 prefix issue. Removed jruby builds in a separate commit.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index da5b8db..40a1517 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,46 +1,26 @@
-jruby:1.7.27-jdk @ amd64
-jruby:1.7.27-jdk @ arm32v6
-jruby:1.7.27-jdk @ arm64v8
-jruby:1.7.27-jdk @ i386
-jruby:1.7.27-jdk @ ppc64le
-jruby:1.7.27-jdk @ s390x
-jruby:1.7.27-jre @ amd64
-jruby:1.7.27-jre @ arm32v6
-jruby:1.7.27-jre @ arm64v8
-jruby:1.7.27-jre @ i386
-jruby:1.7.27-jre @ ppc64le
-jruby:1.7.27-jre @ s390x
-jruby:1.7.27-onbuild @ amd64
-jruby:1.7.27-onbuild @ arm32v6
-jruby:1.7.27-onbuild @ arm64v8
-jruby:1.7.27-onbuild @ i386
-jruby:1.7.27-onbuild @ ppc64le
-jruby:1.7.27-onbuild @ s390x
-jruby:9.1.13.0-jdk @ amd64
-jruby:9.1.13.0-jdk @ arm32v5
-jruby:9.1.13.0-jdk @ arm64v8
-jruby:9.1.13.0-jdk @ i386
-jruby:9.1.13.0-jdk @ ppc64le
-jruby:9.1.13.0-jdk @ s390x
-jruby:9.1.13.0-jdk-alpine @ amd64
-jruby:9.1.13.0-jdk-alpine @ arm32v6
-jruby:9.1.13.0-jdk-alpine @ arm64v8
-jruby:9.1.13.0-jdk-alpine @ ppc64le
-jruby:9.1.13.0-jdk-alpine @ s390x
-jruby:9.1.13.0-jre @ amd64
-jruby:9.1.13.0-jre @ arm32v5
-jruby:9.1.13.0-jre @ arm64v8
-jruby:9.1.13.0-jre @ i386
-jruby:9.1.13.0-jre @ ppc64le
-jruby:9.1.13.0-jre @ s390x
-jruby:9.1.13.0-jre-alpine @ amd64
-jruby:9.1.13.0-jre-alpine @ arm32v6
-jruby:9.1.13.0-jre-alpine @ arm64v8
-jruby:9.1.13.0-jre-alpine @ ppc64le
-jruby:9.1.13.0-jre-alpine @ s390x
-jruby:9.1.13.0-onbuild @ amd64
-jruby:9.1.13.0-onbuild @ arm32v5
-jruby:9.1.13.0-onbuild @ arm64v8
-jruby:9.1.13.0-onbuild @ i386
-jruby:9.1.13.0-onbuild @ ppc64le
-jruby:9.1.13.0-onbuild @ s390x
+jruby:9.1.14.0-jdk @ amd64
+jruby:9.1.14.0-jdk @ arm32v5
+jruby:9.1.14.0-jdk @ arm64v8
+jruby:9.1.14.0-jdk @ i386
+jruby:9.1.14.0-jdk @ ppc64le
+jruby:9.1.14.0-jdk @ s390x
+jruby:9.1.14.0-jdk-alpine @ amd64
+jruby:9.1.14.0-jdk-alpine @ arm64v8
+jruby:9.1.14.0-jdk-alpine @ ppc64le
+jruby:9.1.14.0-jdk-alpine @ s390x
+jruby:9.1.14.0-jre @ amd64
+jruby:9.1.14.0-jre @ arm32v5
+jruby:9.1.14.0-jre @ arm64v8
+jruby:9.1.14.0-jre @ i386
+jruby:9.1.14.0-jre @ ppc64le
+jruby:9.1.14.0-jre @ s390x
+jruby:9.1.14.0-jre-alpine @ amd64
+jruby:9.1.14.0-jre-alpine @ arm64v8
+jruby:9.1.14.0-jre-alpine @ ppc64le
+jruby:9.1.14.0-jre-alpine @ s390x
+jruby:9.1.14.0-onbuild @ amd64
+jruby:9.1.14.0-onbuild @ arm32v5
+jruby:9.1.14.0-onbuild @ arm64v8
+jruby:9.1.14.0-onbuild @ i386
+jruby:9.1.14.0-onbuild @ ppc64le
+jruby:9.1.14.0-onbuild @ s390x
diff --git a/_bashbrew-list b/_bashbrew-list
index fde1faf..9525657 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,13 +1,7 @@
-jruby:1.7
-jruby:1.7-jdk
-jruby:1.7-jre
-jruby:1.7-onbuild
-jruby:1.7.27
-jruby:1.7.27-jdk
-jruby:1.7.27-jre
-jruby:1.7.27-onbuild
 jruby:9
 jruby:9-alpine
+jruby:9-jdk
+jruby:9-jdk-alpine
 jruby:9-onbuild
 jruby:9.1
 jruby:9.1-alpine
@@ -16,18 +10,18 @@ jruby:9.1-jdk-alpine
 jruby:9.1-jre
 jruby:9.1-jre-alpine
 jruby:9.1-onbuild
-jruby:9.1.13
-jruby:9.1.13-alpine
-jruby:9.1.13-jdk
-jruby:9.1.13-jdk-alpine
-jruby:9.1.13-jre
-jruby:9.1.13-jre-alpine
-jruby:9.1.13-onbuild
-jruby:9.1.13.0
-jruby:9.1.13.0-alpine
-jruby:9.1.13.0-jdk
-jruby:9.1.13.0-jdk-alpine
-jruby:9.1.13.0-jre
-jruby:9.1.13.0-jre-alpine
-jruby:9.1.13.0-onbuild
+jruby:9.1.14
+jruby:9.1.14-alpine
+jruby:9.1.14-jdk
+jruby:9.1.14-jdk-alpine
+jruby:9.1.14-jre
+jruby:9.1.14-jre-alpine
+jruby:9.1.14-onbuild
+jruby:9.1.14.0
+jruby:9.1.14.0-alpine
+jruby:9.1.14.0-jdk
+jruby:9.1.14.0-jdk-alpine
+jruby:9.1.14.0-jre
+jruby:9.1.14.0-jre-alpine
+jruby:9.1.14.0-onbuild
 jruby:latest
diff --git a/jruby_1.7.27-jdk/Dockerfile b/jruby_1.7.27-jdk/Dockerfile
deleted file mode 100644
index a5662d3..0000000
diff --git a/jruby_1.7.27-jre/Dockerfile b/jruby_1.7.27-jre/Dockerfile
deleted file mode 100644
index 0ce8ff3..0000000
diff --git a/jruby_1.7.27-onbuild/Dockerfile b/jruby_1.7.27-onbuild/Dockerfile
deleted file mode 100644
index 333fbe3..0000000
diff --git a/jruby_9.1.13.0-jdk-alpine/Dockerfile b/jruby_9.1.14.0-jdk-alpine/Dockerfile
similarity index 92%
rename from jruby_9.1.13.0-jdk-alpine/Dockerfile
rename to jruby_9.1.14.0-jdk-alpine/Dockerfile
index 7cd0fde..5e66b85 100644
--- a/jruby_9.1.13.0-jdk-alpine/Dockerfile
+++ b/jruby_9.1.14.0-jdk-alpine/Dockerfile
@@ -4,8 +4,8 @@ RUN apk add --no-cache \
       bash \
       libc6-compat
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 
 RUN apk add --no-cache --virtual .build-deps \
       curl \
diff --git a/jruby_9.1.13.0-jdk/Dockerfile b/jruby_9.1.14.0-jdk/Dockerfile
similarity index 91%
rename from jruby_9.1.13.0-jdk/Dockerfile
rename to jruby_9.1.14.0-jdk/Dockerfile
index da9e5ad..797b929 100644
--- a/jruby_9.1.13.0-jdk/Dockerfile
+++ b/jruby_9.1.14.0-jdk/Dockerfile
@@ -2,8 +2,8 @@ FROM openjdk:8-jdk
 
 RUN apt-get update && apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 RUN mkdir /opt/jruby \
   && curl -fSL https://s3.amazonaws.com/jruby.org/downloads/${JRUBY_VERSION}/jruby-bin-${JRUBY_VERSION}.tar.gz -o /tmp/jruby.tar.gz \
   && echo "$JRUBY_SHA256 /tmp/jruby.tar.gz" | sha256sum -c - \
diff --git a/jruby_9.1.13.0-jre-alpine/Dockerfile b/jruby_9.1.14.0-jre-alpine/Dockerfile
similarity index 92%
rename from jruby_9.1.13.0-jre-alpine/Dockerfile
rename to jruby_9.1.14.0-jre-alpine/Dockerfile
index 510c6b7..788430c 100644
--- a/jruby_9.1.13.0-jre-alpine/Dockerfile
+++ b/jruby_9.1.14.0-jre-alpine/Dockerfile
@@ -4,8 +4,8 @@ RUN apk add --no-cache \
       bash \
       libc6-compat
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 
 RUN apk add --no-cache --virtual .build-deps \
       curl \
diff --git a/jruby_9.1.13.0-jre/Dockerfile b/jruby_9.1.14.0-jre/Dockerfile
similarity index 91%
rename from jruby_9.1.13.0-jre/Dockerfile
rename to jruby_9.1.14.0-jre/Dockerfile
index 2283b9c..0f7147e 100644
--- a/jruby_9.1.13.0-jre/Dockerfile
+++ b/jruby_9.1.14.0-jre/Dockerfile
@@ -2,8 +2,8 @@ FROM openjdk:8-jre
 
 RUN apt-get update && apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV JRUBY_VERSION 9.1.13.0
-ENV JRUBY_SHA256 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+ENV JRUBY_VERSION 9.1.14.0
+ENV JRUBY_SHA256 074057e672350a6652d92ccaaa5d517fc7d6b980bce8b947515fb64d114d1651
 RUN mkdir /opt/jruby \
   && curl -fSL https://s3.amazonaws.com/jruby.org/downloads/${JRUBY_VERSION}/jruby-bin-${JRUBY_VERSION}.tar.gz -o /tmp/jruby.tar.gz \
   && echo "$JRUBY_SHA256 /tmp/jruby.tar.gz" | sha256sum -c - \
diff --git a/jruby_9.1.13.0-onbuild/Dockerfile b/jruby_9.1.14.0-onbuild/Dockerfile
similarity index 100%
rename from jruby_9.1.13.0-onbuild/Dockerfile
rename to jruby_9.1.14.0-onbuild/Dockerfile
```

</details>

**Comment by yosifkit** [comment]
Build test of #3717; 7a8824fe; `amd64` (`jruby`):

```console

$ bashbrew build jruby:latest
Building bashbrew/cache:5298f947b8ebcd84c41105fddc5ac59d4599e344d15f314ac132f90fcb9213b8 (jruby:latest)
Tagging jruby:latest
Tagging jruby:9
Tagging jruby:9.1
Tagging jruby:9.1.14
Tagging jruby:9.1-jre
Tagging jruby:9.1.14-jre
Tagging jruby:9.1.14.0
Tagging jruby:9.1.14.0-jre

$ test/run.sh jruby:latest
testing jruby:latest
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...passed
	'override-cmd' [4/9]...passed
	'ruby-hello-world' [5/9]...passed
	'ruby-standard-libs' [6/9]...passed
	'ruby-gems' [7/9]...passed
	'ruby-bundler' [8/9]...passed
	'ruby-nonroot' [9/9]...passed


$ bashbrew build jruby:9-alpine
Building bashbrew/cache:913204b613e2b0c68fe177f46481d640fc2ba7bb2ff4282d061d7809b306cf0a (jruby:9-alpine)
Tagging jruby:9-alpine
Tagging jruby:9.1-alpine
Tagging jruby:9.1.14-alpine
Tagging jruby:9.1-jre-alpine
Tagging jruby:9.1.14-jre-alpine
Tagging jruby:9.1.14.0-alpine
Tagging jruby:9.1.14.0-jre-alpine

$ test/run.sh jruby:9-alpine
testing jruby:9-alpine
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'ruby-hello-world' [5/7]...passed
	'ruby-standard-libs' [6/7]...passed
	'ruby-nonroot' [7/7]...passed


$ bashbrew build jruby:9-jdk
Building bashbrew/cache:604d916ed09844dfde13b0487e4c899299255dd44fb273a046a480cdf80a2ed7 (jruby:9-jdk)
Tagging jruby:9-jdk
Tagging jruby:9.1-jdk
Tagging jruby:9.1.14-jdk
Tagging jruby:9.1.14.0-jdk

$ test/run.sh jruby:9-jdk
testing jruby:9-jdk
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...passed
	'override-cmd' [4/9]...passed
	'ruby-hello-world' [5/9]...passed
	'ruby-standard-libs' [6/9]...passed
	'ruby-gems' [7/9]...passed
	'ruby-bundler' [8/9]...passed
	'ruby-nonroot' [9/9]...passed


$ bashbrew build jruby:9-jdk-alpine
Building bashbrew/cache:dae453b4a0e68bf04a053141328011473ff092bf79aa5be4bde061956a645e79 (jruby:9-jdk-alpine)
Tagging jruby:9-jdk-alpine
Tagging jruby:9.1-jdk-alpine
Tagging jruby:9.1.14-jdk-alpine
Tagging jruby:9.1.14.0-jdk-alpine

$ test/run.sh jruby:9-jdk-alpine
testing jruby:9-jdk-alpine
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'ruby-hello-world' [5/7]...passed
	'ruby-standard-libs' [6/7]...passed
	'ruby-nonroot' [7/7]...passed


$ bashbrew build jruby:9-onbuild
Building bashbrew/cache:f250c3d2998f8777f19b384854cada3c5a2d20f03c596b87958f06b8f584f3c0 (jruby:9-onbuild)
Tagging jruby:9-onbuild
Tagging jruby:9.1-onbuild
Tagging jruby:9.1.14-onbuild
Tagging jruby:9.1.14.0-onbuild

$ test/run.sh jruby:9-onbuild
testing jruby:9-onbuild
	'override-cmd' [1/1]...passed

```

**Pull request merged**